packages/core/primitives/event-dispatch/src/event.ts

```diff
@@ -79,7 +84,13 @@ export function addEventListener(
  */
 export function removeEventListener(element: Element, info: EventHandlerInfo) {
   if (element.removeEventListener) {
-    element.removeEventListener(info.eventType, info.handler as EventListener, info.capture);
+    if (typeof info.passive === 'boolean') {
+      element.removeEventListener(info.eventType, info.handler as EventListener, {
+        capture: info.capture,
+      });
+    } else {
+      element.removeEventListener(info.eventType, info.handler as EventListener, info.capture);
+    }
```

Comment:
I'd propose a similar refactoring here + also add a comment on why we pass capture like this (we can use a quote from https://github.com/angular/angular/pull/58198/files#r1809081367):
```suggestion
    const options = typeof info.passive === 'boolean' ? {capture: info.capture} : info.capture;
    element.removeEventListener(info.eventType, info.handler as EventListener, options);
```

Reply:
Done.